src/service/advanced-search/search.service.ts

```diff
@@ -63,7 +63,7 @@ export const getSearchResults = async (advancedSearchParams: AdvancedSearchParam
                 {
                     html: `<h2 class="govuk-heading-m" style="margin-bottom: 3px;"><a class="govuk-link" href=${links.company_profile} target="_blank">${company_name}<span class="govuk-visually-hidden">(link opens a new window)</span></a></h2>
                             <p style="margin-top:0px;">
-                            <span class="govuk-body govuk-!-font-weight-bold">${mappedCompanyStatus}</span></p>
+                            ${mappedCompanyStatus}
```

Comment:
this should have the closing </p> tag I believe?

Reply:
Yeah sorry, i've updated this now as I didn't correctly implement the new version without the breakpoints. Fix should now be present.